docs/revsets.md

```diff
@@ -164,6 +164,7 @@ You can use parentheses to control evaluation order, such as `(x & y) | z` or
     **Operator** `x::y`
 
     * `D::D` ⇒ `{D}`
+    * `B::C` ⇒ `{}` (empty set) (note that, unlike `B..C`, this excludes `C`)
```

Comment:
(Optional) I think putting `B::C` after `B::D` and `B..C` after `B..D` might be slightly better from the pedagogic perspective. Understanding the behavior of `B..D` might help understand `B..C`.

But it is less alphabetical. 🤷

Reply:
Done. It's actually more consistently reverse alphabetical :)